rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -2125,7 +2125,7 @@ public void setBusyNode(final String nodeUrl, Client owner) throws NotConnectedE
             return;
         }
 
-        if (!clients.containsKey(owner.getId())) {
+        if ((!clients.containsKey(owner.getId()) && owner != localClient)) {
```

Comment:
There is an extra parenthesis nest.
Otherwise I agree for adding this condition.